signing_service/tests/tests.py

```diff
@@ -0,0 +1,4 @@
+
+
+def test_that_full_check_will_run_this_test():
+    assert True
```

Comment:
Actually there could be some unit tests for the following cases:

- If connection is interrupted due to a failure, it should try to reconnect.
- If connection is closed by other side, exit gracefully.
- If a shutdown signal or KeyboardInterrupt is caught, exit gracefully.
- If there was an unrecognized exception, it logs it and report to Sentry, then reraise and crash.

Reply:
I thought no one will notice ;) Yes, most of this can be tested but a good integration tests are not possible in this case I think. But I will add those which are possible.

Reply:
at least some unit tests with mocks ;)

Reply:
Do you mean tests that will check communication between SigningService, Middleman and Concent ? Something like our `cluster` tests ? I will add the issue but please clarify this is what should be in it.

Reply:
Created https://github.com/golemfactory/concent/issues/647

> Though if you can think of any good way to prepare integration tests like the ones we had initially for Concent (running on Django's test case) that would be nice too.

I think it might be done but when all the components get more mature.